dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/EditDataset.ts

```diff
@@ -1,82 +0,0 @@
-import { getKeycloakToken } from "@e2e/utils/Auth";
-import { admin_name, admin_pw, uploader_name, uploader_pw } from "@e2e/utils/Cypress";
-import { DataTypeEnum, type LksgData } from "@clients/backend";
-import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
-import { describeIf } from "@e2e/support/TestUtility";
-import { submitButton } from "@sharedUtils/components/SubmitButton";
-import { type UploadIds } from "@e2e/utils/GeneralApiUtils";
-import { uploadCompanyAndFrameworkData } from "@e2e/utils/FrameworkUpload";
-
-describeIf(
-  "Validates the edit button functionality on the view framework page",
-  {
-    executionEnvironments: ["developmentLocal", "ci"],
-  },
-  () => {
-    let uploadIds: UploadIds;
-    let lksgFixture: FixtureData<LksgData>;
-
-    before(() => {
-      cy.fixture("CompanyInformationWithLksgPreparedFixtures").then(function (jsonContent) {
-        const lksgsPreparedFixtures = jsonContent as Array<FixtureData<LksgData>>;
-        lksgFixture = getPreparedFixture("lksg-all-fields", lksgsPreparedFixtures);
-        lksgFixture.companyInformation.identifiers = {};
-        getKeycloakToken(admin_name, admin_pw)
-          .then(async (token: string) =>
-            uploadCompanyAndFrameworkData(
-              DataTypeEnum.Lksg,
-              token,
-              lksgFixture.companyInformation,
-              lksgFixture.t,
-              lksgFixture.reportingPeriod,
-            ),
-          )
-          .then((idsUploaded) => {
-            uploadIds = idsUploaded;
-          });
-      });
-    });
-
-    it("Edit and subsequent upload should work properly when removing or changing referenced documents", () => {
-      cy.ensureLoggedIn(uploader_name, uploader_pw);
-      cy.visitAndCheckAppMount(
```

Comment:
@fsoffel Überprüfen wir jetzt noch irgendwo, dass das editieren von referenzierten Dokumenten noch funktioniert?

Reply:
Ja tun wir in YesNoBaseDataPointFormField.cy.ts